test/shared/hateoas/MockHateoasFactory.scala

```diff
@@ -32,11 +33,11 @@ trait MockHateoasFactory extends MockFactory {
         .expects(a, data, *)
     }
 
-    def wrapList[A[_]: Functor, I, D <: HateoasData](a: A[I], data: D): CallHandler[HateoasWrapper[A[HateoasWrapper[I]]]] = {
-      (mockHateoasFactory
-        .wrapList(_: A[I], _: D)(_: Functor[A], _: HateoasListLinksFactory[A, I, D]))
-        .expects(a, data, *, *)
-    }
+    // def wrapList[A[_]: Functor, I, D <: HateoasData](a: A[I], data: D): CallHandler[HateoasWrapper[A[HateoasWrapper[I]]]] = {
+    //   (mockHateoasFactory
+    //     .wrapList(_: A[I], _: D)(_: Functor[A], _: HateoasListLinksFactory[A, I, D]))
+    //     .expects(a, data, *, *)
+    // }
```

Comment:
This mock might need some work in other APIs where it is used, although didn't spend time on it as HATEOAS is being removed

Reply:
See this commit https://github.com/hmrc/business-details-api/commit/86439134ee0d51fa34d91f234520cf6a2e4265ef for replacing mock requirements for services yet to remove HATEOAS